genesyscloud/station/genesyscloud_station_init_test.go

```diff
@@ -0,0 +1,75 @@
+package station
+
+import (
+	"sync"
+	"testing"
+
+	gcloud "terraform-provider-genesyscloud/genesyscloud"
+	edgePhone "terraform-provider-genesyscloud/genesyscloud/telephony_providers_edges_phone"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+/*
+   The genesyscloud_station_init_test.go file is used to initialize the data sources and resources
+   used in testing the station.
+
+   Please make sure you register ALL resources and data sources your test cases will use.
+*/
+
+const (
+	trueValue  = "true"
+	falseValue = "false"
+	nullValue  = "null"
+)
```

Comment:
I see us using these values a lot, it might be worth creating them as constants in the `genesyscloud` package to save use replicating them in every package

Reply:
I agree about putting it inside gcloud package but could do with a separate PR. Just want to keep the PR to the minimum required change to implement feature.